test/tests/S6ImportExportTests.cpp

```diff
@@ -120,7 +120,7 @@ static bool ExportSave(MemoryStream& stream, std::unique_ptr<IContext>& context)
     exporter->ExportObjectsList = objManager.GetPackableObjects();
 
     auto& gameState = getGameState();
-    exporter->Export(gameState, stream);
+    exporter->Export(gameState, stream, 7);
```

Comment:
Magic number, should have a constant.

Reply:
I added a constant to the test file itself. Not sure if that's the best solution, so I'm gonna keep this marked as unresolved for the time being.